rfc/0001-checks-customization.md

```diff
@@ -0,0 +1,325 @@
+|              |                                                      |
+| :----------- | :--------------------------------------------------- |
+| Feature Name | Checks Customization                                 |
+| Start Date   | Jan 15th, 2025                                       |
+| Category     | Architecture                                         |
+
+# Summary
+
+Certain checks in the Trento checks catalog should be customized by customers on a target basis to adjust the corresponding expected values to ones that work better in their environment.
+
+### Use Cases outline
+
+- As a user, I want to override a Check's expected values on a target basis (host or cluster at the moment) so that it matches my specific environment needs
+- As a user, I want the overridden values to be used in a checks execution
+- As a user, I want to reset a cutomized check to default so that I can return to using SUSE's suggested best practices
```

Comment:
Typo in: cutomized

Reply:
Fixed https://github.com/trento-project/docs/pull/62/commits/1340ca57985c71e6ba8f29db40195b740e752db5